GameData/RealismOverhaul/Engine_Configs/F1_Config.cfg

```diff
@@ -168,7 +170,7 @@
 			description = Uprated and simplified F-1 developed for post-Apollo launch vehicles. Tested extensively, but cancelled following the cancellation of Apollo
 			minThrust = 7855.6
 			maxThrust = 9189.6
-			massMult = 0.97673
+			massMult = 1.02677
```

Comment:
Sutton describes the F-1A as a "larger version of F-1" but no exact weight info, so I checked other sources and found this Rocketdyne Division document via alternatewars: https://www.alternatewars.com/BBOW/Space_Engines/F-1A_Specs.gif
Slightly off from this exact document by ~100 lbs, but, it is described as (est), so that can be changed/up for debate.

Reply:
That is from a 90s study, not the original early 70s F-1A. It might be a completely different, F-1 derived engine named the F-1A?

Reply:
Oh that makes sense. Should have realized something was up when it said "Rockwell International" rather than NAR.

Reply:
Weird. Well, if everything that NK can find says the F-1A was lighter, I'll revert the massMult.

Reply:
I would just revert the F-1A changes entirely since they seem based on a different F-1A? And then we can dig later to find the 60s-era F-1A changes.